openmetadata-service/src/main/java/org/openmetadata/service/search/elasticsearch/ElasticSearchClient.java

```diff
@@ -2322,7 +2308,13 @@ public Object getLowLevelClient() {
 
   private void buildSearchRBACQuery(
       SubjectContext subjectContext, SearchSourceBuilder searchSourceBuilder) {
-    if (subjectContext != null && !subjectContext.isAdmin()) {
+    if (Boolean.TRUE.equals(
```

Comment:
@mohityadav766 lets create a common method for check this, given we have it in both

Reply:
okay @harshach

Reply:
resolved